en/docs/administer/managing-users-and-roles/managing-user-roles.md

```diff
@@ -109,7 +109,7 @@ Follow the instructions below to create the `creator`, `publisher`, and `subscri
     These roles do not have any permissions assigned to it, but it is used to manage the visibility of the corresponding service provider that is created in the format of `'<username>_<applicationName>_PRODUCTION'` within the Key Manager. The created service provider is only visible to users with the latter mentioned role that has been generated automatically. Only if a user with admin privileges assigns the latter mentioned role to a user, will that user be able to view the details of the service provider that is created per application.
 
 !!! warning
-    As a limitation, when you delete a user and create another with the same username, applications of the previous user will be visible on the Dev portal applications listing page.
+    As a limitation, when you delete a user and create another with the same username, applications of the previous user will be visible on the Developer portal applications listing page.
```

Comment:
Developer portal > Developer Portal

Reply:
Portal is correctly spelt so the tool does not flag it. Will fix manually.

Reply:
Fixed.